scripts/install_requirements.sh

```diff
@@ -3,4 +3,5 @@
 pip install -r requirements.txt
 python -m nltk.downloader punkt
 python -m spacy.en.download all
-pip install --no-cache-dir http://download.pytorch.org/whl/cu80/torch-0.1.11.post5-cp35-cp35m-linux_x86_64.whl
\ No newline at end of file
+pip install --no-cache-dir http://download.pytorch.org/whl/cu80/torch-0.1.11.post5-cp35-cp35m-linux_x86_64.whl
+conda install pytorch torchvision -c soumith
```

Comment:
PyTorch doesn't support pip fully yet.  Running `pip install pytorch` opens a web browser to the PyTorch homepage--they actually changed the behavior yesterday I think--In the morning it output a prompt.

It's unfortunate that we use both conda and pip in the same project.